components/subscribe-to-stanford/Component.jsx

```diff
@@ -35,7 +33,7 @@ export default function SubscribeToStanfordReport({ contentConfiguration }) {
     if (!email || !email.match(/\w+@\w+\.\w+/)) {
       setSubscriptionError(
         () =>
-          `<span class="su-block su-text-16 su-font-normal su-text-digital-red">Please enter a valid email address.</span>`
+          `<span id="subscription-error" class="su-block su-text-16 su-font-normal su-text-digital-red-light">Please enter a valid email address.</span>`
```

Comment:
Error message needs an `id`, which will be linked to by the input field using the `aria-describedby` attribute. We had to do this everywhere in the ADAPT MyAccount project so I thought I'd do it here as well.